pkgs/applications/office/beancount/bean-add.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchgit, pkgs, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  version = "2016-09-29-76945c320e6f028223e4420956c80c421c4fe74a";
+  name = "bean-add-${version}";
+  namePrefix = "";
+
+  src = fetchgit {
+    url = "https://github.com/simon-v/bean-add/";
+    rev = "76945c320e6f028223e4420956c80c421c4fe74a";
+    sha256 = "0dslzaimfwqvwz4z1qlmjvr5gklfd9n6hbsghl375ndfj8qgql1y";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ readline ];
+
+  phases = [ "unpackPhase" "patchPhase" "installPhase" "fixupPhase" ];
+
+  installPhase = ''
+    # patchShebangs bean-add does not work or I am too dumb to use it
+    sed -i "s,/usr/bin/python,/usr/bin/env python," bean-add
```

Comment:
shebangs will have been patched, except if they're strings somewhere in the files.